### PR TITLE
Use FetchPolicy in useLazyLoadQuery

### PR DESCRIPTION
**Pull request by iukondra**

### CODE REVIEW

File: packages/nova-react/src/graphql/types.ts [hunk 1]
```diff
@@ -82,11 +82,17 @@ export type PaginationFn = (
...
 ) => Disposable;
 
-type FetchPolicy =
+export type FetchPolicy =
   | "store-or-network"
   | "store-and-network"
   | "network-only"
-  | "store-only";
+  | "store-only"
+  | "cache-first"
```
Comment (sjwilczynski) [inline packages/nova-react/src/graphql/types.ts:90]: What are the new fetch policies? Are there from apollo-client? I think we should just use Relay ones as [duct tape does translation](https://github.com/microsoft/graphitation/blob/main/packages/apollo-react-relay-duct-tape/src/convertFetchPolicy.ts). Right, @alloy ?

Comment (iukondra) [reply packages/nova-react/src/graphql/types.ts:90]: Apollo fetch policies. If we don't need to support them, I'm happy to remove them

Comment (alloy) [reply packages/nova-react/src/graphql/types.ts:90]: Yes, only the Relay ones, and specifically the ones captured in the duct-tape transform 👍

**Review by dragoshomner — APPROVED** [review]
Looks good to me. Please wait Eloy's review before completing it.

**Review by sjwilczynski — APPROVED** [review]

**Review by alloy — APPROVED** [review]

**Pull request merged**